recipes/rb-connection_pool/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "connection_pool" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 677985be912f33c90f98f229aaa0c0ddb2ef8776f21929a36eeeb25251c944da
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+    - cp {{ name }}-*/LICENSE .
+  skip: true  # [win]
+
+requirements:
+  host:
+    - ruby <2.6
+  run:
+    - ruby <2.6
```

For now, let's pin ruby more explicit.
```suggestion
    - ruby >=2.5,<2.6
  run:
    - ruby >=2.5,<2.6
```